.github/workflows/prepare-release.yml

```
@@ -1,35 +1,44 @@
   prepare:
-    if: ${{ inputs.confirmation == 'RELEASE 0.18.1' }}
+    if: ${{ inputs.confirmation == format('RELEASE {0}', inputs.version) }}
     runs-on: ubuntu-latest
     timeout-minutes: 35
     env:
-      RELEASE_VERSION: 0.18.1
-      RELEASE_BRANCH: release/v0.18.1
-      RELEASE_TITLE: "chore(release): v0.18.1"
+      RELEASE_VERSION: ${{ inputs.version }}
+      RELEASE_BRANCH: ${{ inputs.branch || format('release/v{0}', inputs.version) }}
+      RELEASE_TITLE: ${{ inputs.title || format('chore(release): v{0}', inputs.version) }}
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)